ui/app/controllers/jobs/run/templates/new.js

```diff
@@ -19,6 +22,17 @@ export default class JobsRunTemplatesController extends Controller {
     return namespaces;
   }
 
+  get isDuplicateTemplate() {
+    const templates = this.store.peekAll('variable');
```

Comment:
Suggestion: either use the queried variables from the route's model at jobs/run/templates/new, or else alias this somewhere outside the isDuplicateTemplate() getter. As it stands, this is going to make a store.peekAll() request every time you type a character into the path input element, which is probably doing more work than it needs to.

Reply:
Great callout! I'll address this in the `edit` template PR!